Makefile

```diff
@@ -176,7 +176,7 @@ bundle: manifests kustomize ## Generate bundle manifests and metadata, then vali
 
 .PHONY: bundle-build
 bundle-build: ## Build the bundle image.
-	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
+	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) . --load
```

Comment:
No need for --load since buildx is not being used; `docker build` automatically stores the built image in the local Docker image store.

Reply:
i think podman uses buildx internally.. can't build with out --load when using podman desktop